src/content/concepts/mode.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Mode
+sort: 4
+contributors:
+  - EugeneHlushko
+---
+
+Providing the `mode` configuration option tells webpack to use its built-in optimizations accordingly.
+
+`string`
+
+## Usage
+
+Just provide the `mode` option in the config:
+
+```javascript
+module.exports = {
+  mode: 'production'
+};
+```
+
+ 
+or pass it as a cli argument:
+
+```bash
+webpack --mode=production
+```
+
+The following string values are supported:
```

Comment:
We should add a note or something clarifying that setting `NODE_ENV` doesn't set `mode` automatically, some users are confused by this: https://github.com/webpack/webpack/issues/2537#issuecomment-370384223